src/commands/Fun Commands/RiddleCommand.ts

```diff
@@ -0,0 +1,19 @@
+import { RunFunction } from '../../interfaces/Command';
+import fetch from 'node-fetch';
+
+export const run: RunFunction = async (client, message, args) => {
+	const response: { question: string; answer: string } = await fetch(
+		'https://no-api-key.com/api/v1/riddle'
+	).then((res) => res.json());
+
+	return message.channel.send(
+		client
+			.embed({ title: 'Riddle!' }, message)
+			.addField(response.question, `||${response.answer}||`)
+	);
+};
+
+export const name: string = 'riddle';
+export const category: string = 'fun';
+export const description: string =
+	'Fetch a random fact and answer from an API.';
```

Comment:
update description

Reply:
linting did that let me fix

Reply:
Linting? I'm talking about the actual content of the description... Shouldn't that be changed?

Reply:
Oh lmao ok